gsuite-device-update-notifier/main.js

```diff
@@ -1,8 +1,8 @@
 var MINIMUM_ANDROID_VERSION = new Date(2019, 8, 5);
 var ANDROID_LAST_UPDATE = new Date(2019, 8, 3);
 
-var MINIMUM_IOS_VERSION = splitOSVersion("13.1");
-var IOS_LAST_UPDATE = new Date(2019, 8, 24);
+var MINIMUM_IOS_VERSION = splitOSVersion("13.1.2");
```

Comment:
Thanks @paultag!

I'm on the fence on whether the right version to put here is `13.1.1` or `13.1.2` -- as far as I can tell, 13.1.2 does not contain any vulnerability fixes. Since the primarily motivation in tracking these is security, placing `13.1.1` as the minimum version (along with a comment) seems appropriate.

Reply:
I've bumped it for myself, but only because I'm trying to make sure I keep the muscle memory strong and used regularly; but I can see not bumping it to create artificial churn, which would lead to people in the org starting to tune out someone asking them to upgrade

Reply:
Ok, I'm compelled by this (sorry about the delay, was OOO for Rosh Hashanah)